packages/UI/src/components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,111 @@
+import {
+  ChevronsLeft,
+  ChevronsRight,
+  ChevronRight,
+  ChevronLeft,
+} from 'lucide-react';
+import Link from 'next/link';
+import * as React from 'react';
+
+import type { VariantProps } from 'tailwind-variants';
+import { tv } from 'tailwind-variants';
+import { cn } from '../../utils/classesMerger';
+
+const Pagination = ({ className, ...props }: React.ComponentProps<'nav'>) => (
+  <nav
+    role="navigation"
+    aria-label="pagination"
+    className={cn('mx-auto flex w-full justify-center', className)}
+    {...props}
+  />
+);
+Pagination.displayName = 'Pagination';
+
+const PaginationContent = React.forwardRef<
+  HTMLUListElement,
+  React.ComponentProps<'ul'>
+>(({ className, ...props }, ref) => (
+  <ul
+    ref={ref}
+    className={cn('flex flex-row items-center gap-8', className)}
+    {...props}
+  />
+));
+PaginationContent.displayName = 'PaginationContent';
+
+const PaginationItem = React.forwardRef<
+  HTMLLIElement,
+  React.ComponentProps<'li'>
+>(({ className, ...props }, ref) => (
+  <li ref={ref} className={cn('', className)} {...props} />
+));
+PaginationItem.displayName = 'PaginationItem';
+
+const paginationLinkStyles = tv({
+  base: 'inline-flex flex-col items-start gap-2,5 rounded-lg px-4 py-2 font-nunito-sans text-base font-normal leading-6',
```

Comment:
gap-2,5

Reply:
that's the alternative we use for `gap-2.5` coming from tailwind.
it's the value of gap between `gap-2` and `gap-3`.
I wish we were able to have with a dot rather than comma but Tokens-studio doesn't accept dots in the name of the toke